src/components/EmailsList.tsx

```diff
@@ -0,0 +1,217 @@
+      text: "text-text",
+      background: "text-background",
+      foreground: "text-foreground",
+    },
+    textTransform: {
+      none: "",
+      uppercase: "ve-uppercase",
+      lowercase: "ve-lowercase",
+      capitalize: "ve-capitalize",
+    },
+    padding: {
+      default: "px-4 py-16 md:px-8",
+      none: "p-0",
+      small: "px-4 py-8 md:px-8",
+      large: "px-[200px] py-24 md:px-8",
+    },
+  },
+  defaultVariants: {
+    padding: "none",
+    fontWeight: "default",
+    color: "default",
+    textTransform: "none",
+  },
+});
+
+export interface EmailListProps extends VariantProps<typeof emailListVariants> {
+  list: YextEntityField<string[]>;
+  includeHyperlink: boolean;
+  listLength: number;
+}
+
+const EmailListFields: Fields<EmailListProps> = {
+  list: YextEntityFieldSelector({
+    label: "Entity Field",
+    filter: {
+      types: ["type.string"],
+      allowList: ["emails"],
+      includeListsOnly: true,
+    },
+  }),
+  fontSize: {
+    label: "Font Size",
+    type: "select",
+    options: [
+      { label: "Extra Small", value: "xs" },
+      { label: "Small", value: "sm" },
+      { label: "Medium", value: "medium" },
+      { label: "Large", value: "lg" },
+      { label: "Extra Large", value: "xl" },
+      { label: "2xl", value: "2xl" },
+      { label: "3xl", value: "3xl" },
+      { label: "4xl", value: "4xl" },
+      { label: "5xl", value: "5xl" },
+    ],
+  },
+  fontWeight: {
+    label: "Font Weight",
+    type: "select",
+    options: [
+      { label: "Default", value: "default" },
+      { label: "Thin", value: "thin" },
+      { label: "Extra Light", value: "extralight" },
+      { label: "Light", value: "light" },
+      { label: "Normal", value: "normal" },
+      { label: "Medium", value: "medium" },
+      { label: "Semibold", value: "semibold" },
+      { label: "Bold", value: "bold" },
+      { label: "Extrabold", value: "extrabold" },
+      { label: "Black", value: "black" },
+    ],
+  },
+  padding: {
+    label: "Padding",
+    type: "radio",
+    options: [
+      { label: "None", value: "none" },
+      { label: "Small", value: "small" },
+      { label: "Medium", value: "default" },
+      { label: "Large", value: "large" },
+    ],
+  },
+  color: {
+    label: "Color",
+    type: "select",
+    options: [
+      { label: "Default", value: "default" },
+      { label: "Primary", value: "primary" },
+      { label: "Secondary", value: "secondary" },
+      { label: "Accent", value: "accent" },
+      { label: "Text", value: "text" },
+      { label: "Background", value: "background" },
+    ],
+  },
+  textTransform: {
+    label: "Text Transform",
+    type: "select",
+    options: [
+      { label: "None", value: "none" },
+      { label: "Uppercase", value: "uppercase" },
+      { label: "Lowercase", value: "lowercase" },
+      { label: "Capitalize", value: "capitalize" },
+    ],
+  },
+  includeHyperlink: {
+    label: "Include Hyperlink",
+    type: "radio",
```

Comment:
Is there any kind of "checkbox" type in Puck? I think that's better for a boolean value if it exists.

Reply:
Nope, these are the options: https://puckeditor.com/docs/api-reference/fields